cabal.project

```diff
@@ -104,62 +77,15 @@ package cardano-wallet-launcher
 package cardano-wallet-core-integration
   optimization: False
 
--- Direct dependency.
--- Compared to others, cardano-wallet doesn't bump dependencies very often.
--- Making it a good place to start when bumping dependencies.
--- As, for example, bumping the node first highly risks breaking API with the wallet.
--- Unless early bug fixes are required, this is fine as the wallet tracks stable releases of the node.
--- And it is indeed nice for marconi track stable releases of the node too.
---
--- The current version is dated 2022/08/10
-source-repository-package
-    type: git
-    location: https://github.com/input-output-hk/cardano-wallet
-    tag: 18a931648550246695c790578d4a55ee2f10463e
-    subdir:
-      lib/text-class
-
--- Should follow cardano-wallet.
-source-repository-package
-    type: git
-    location: https://github.com/input-output-hk/cardano-addresses
-    tag: b7273a5d3c21f1a003595ebf1e1f79c28cd72513
-    subdir:
-      -- cardano-addresses-cli
-      command-line
-      -- cardano-addresses
-      core
-
--- This is needed because we rely on an unreleased branch of
--- cardano-ledger-alonzo. The feature we need
--- (evaluateTransactionExecutionUnitsWithLogs) in only included from
--- cardano-ledger-alonzo version 1.0.0.0
--- See https://github.com/input-output-hk/cardano-ledger/pull/3111
 source-repository-package
```

Comment:
This is for the unofficial 8.0.0 release? Maybe a comment saying that is the case, and say it would need to use the CHaP release once available.